UPchieve/web
# Student waiting long

Links
-----
- Server repo PR: https://github.com/UPchieve/server/pull/505

Description
-----------
Both cases: show a modal if a student has been waiting too long for a volunteer to join the session
- After 15 minutes, a modal saying that we are having trouble matching their session with prompts to exit the modal or end their session
- After 45 minutes, a modal saying that we could not match their session with a button to end the session


Steps to test:
1. Start a session
2. Wait 15 minutes
   2a. Or change the `createdAt` time on the session in MongoDB to be more than 15 minutes ago but less than 45
   2b. Or change `fifteenMinsFromSessionStart` in `SessionHeader.vue` to a shorter time to test quicker
3. A modal should appear saying that there’s trouble getting a volunteer for the session
4. Cancel the modal
4. Do steps 2 and 3, but now with a 45 minute wait period
5. A modal should appear saying that we couldn’t find a volunteer to match the student and the modal will prompt the student to end the session

Developer self-review checklist
-------------------------------
